rental.rb

```diff
@@ -0,0 +1,15 @@
+require 'date'
+
+class Rental
+  attr_accessor :date
+
+  def initialize(person, book, date = DateTime.now())
+    @date = date
+
+    @person = person
+    person.rentals << self
+
+    @book = book
+    book.rentals << self
+  end
+end
```

Comment:
The rental class is good, no need to change any code here. Since you will be creating new instances of Rental on both Book and Person, whenever the method is called from them, the initialize here will make sure to include itself on the book.rentals and person.rentals! Good job here! :clap: